javascript/javascript1/week4/hw4.js

```diff
@@ -0,0 +1,86 @@
+let todo = [];
+let myName = '';
+const month = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+function getReply(command) {
+
+    if (command.includes('Hello my name is Hamed Hello my name is Hamed')) {
+        return 'Please tell me just once your information';
+
+    } else if (command === 'Hello my name is Hamed') {
+        myName = command.substring(18);
```

Comment:
Can you try to split the command and get the index of the name, in that case you can get any name at the end.